include/ada/url_base.h

```diff
@@ -35,94 +37,13 @@ namespace ada {
      */
     ada::scheme::type type{ada::scheme::type::NOT_SPECIAL};
 
-    /** @see https://url.spec.whatwg.org/#dom-url-username */
```

Comment:
The documentations are gone with these changes. I think we should make these all virtual, and keep the comments in `url_base`